### PR TITLE
Update README

### PR DESCRIPTION
**Pull request by ericbottard**

### CODE REVIEW

File: README.adoc [hunk 1]
```diff
@@ -1,3 +1,121 @@
-Spring Cloud Data Flow Admin for Cloud Foundry
-----------------------------------------------
+= Spring Cloud Data Flow Admin for Cloud Foundry
+
+Spring Cloud Data Flow can be used to deploy modules in a Cloud Foundry environment. When doing so, the
+https://github.com/spring-cloud/spring-cloud-dataflow/tree/master/spring-cloud-dataflow-admin[Admin]
```
Comment (markfisher) [inline README.adoc]: this link is no longer valid

File: README.adoc [hunk 2]
```diff
@@ -1,3 +1,121 @@
...
+```
+
+3. Either
+	a. push the admin application on Cloud Foundry, configure it (see below) and start it
++
+NOTE: You must use a unique name for your app; an app with the same name in the same organization will cause your
+deployment to fail
++
+```
+cf push s-c-dataflow-admin --no-start -p spring-cloud-dataflow-admin/target/spring-cloud-dataflow-admin-1.0.0.BUILD-SNAPSHOT.jar
```
Comment (markfisher) [inline README.adoc]: this is no longer correct

File: README.adoc [hunk 3]
```diff
@@ -1,3 +1,121 @@
...
+```
+cf start s-c-dataflow-admin
+```
++
+Alternatively,
++
+	b. run the admin application locally, targeting your Cloud Foundry installation (see below for configuration)
++
+```
+java -jar spring-cloud-dataflow-admin/target/spring-cloud-dataflow-admin-1.0.0.BUILD-SNAPSHOT.jar [--option1=value1] [--option2=value2] [etc.]
```
Comment (markfisher) [inline README.adoc]: update this also

File: README.adoc (hunk 1 above)
Comment (ericbottard) [inline README.adoc]: Ah, good point. I got carried away by asciidoc conversion, did not actually read what I was copy/pasting. Will update

**Comment by ericbottard** [comment]
Updated according to new project names. We may also want to extract the version numbers in an asciidoc attribute, but this should
* be consistent across all projects
* require usage of asciidoc across all projects
IMO

**Comment by markfisher** [comment]
One of the key benefits of isolated admin implementations is that their deployment lifecycles can be independent. That includes the ability to have different version numbers (i.e. it need not match the version of its dependency on the dataflow admin starter).

**Comment by markfisher** [comment]
LGTM; squashed and merged as https://github.com/spring-cloud/spring-cloud-dataflow-admin-cloudfoundry/commit/34956b959804d3b5a8ada76042e4e61059b3c19e

**Comment by ericbottard** [comment]
What I meant was extracting the version number of *this* project so that it does not have to be hardcoded and repeated in the README. Definitely not that it would be kept in sync with the other project's

**Comment by markfisher** [comment]
gotcha and +1

**Pull request closed without merging**